### PR TITLE
FrozenLake tutorial

### PR DESCRIPTION
**Pull request by kir0ul**
# Description

This PR adds a benchmark tutorial on FrozenLake, related to issue https://github.com/Farama-Foundation/Gymnasium/issues/28.

## Type of change

Please delete options that are not relevant.

- [x] Change to the documentation

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kir0ul** [comment]
First time I write a tutorial, so I'm happy to take any suggestion to improve it!

File: docs/tutorials/training_agents/FrozenLake_tuto.py [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+        "seed",
+        "is_slippery",
+        "n_runs",
+        "action_size",
+        "state_size",
+        "proba_frozen",
+        "savefig_folder",
+    ],
+)
+p = Params(
```
Comment (pseudo-rnd-thoughts) [inline docs/tutorials/training_agents/FrozenLake_tuto.py:67]: Could you rename to `params` to so it is easier to remember compared to `p`

**Review by pseudo-rnd-thoughts — CHANGES_REQUESTED** [review]
This looks amazing. Thank you for doing this
Just two small comments to help with readability

File: docs/tutorials/training_agents/FrozenLake_tuto.py (hunk 1 above)
Comment (kir0ul) [reply docs/tutorials/training_agents/FrozenLake_tuto.py]: Done in https://github.com/Farama-Foundation/Gymnasium/pull/376/commits/21e494fee066dcee2f79207e13d4b6bc82cfb4ea.

File: docs/tutorials/training_agents/FrozenLake_tuto.py [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+
+# %load_ext lab_black
+
+
+# %%
+# Parameters we'll use
+# --------------------
+#
+
+Params = namedtuple(
```
[the remark this answers is not in the document]
Comment (kir0ul) [reply docs/tutorials/training_agents/FrozenLake_tuto.py]: Done in https://github.com/Farama-Foundation/Gymnasium/pull/376/commits/21e494fee066dcee2f79207e13d4b6bc82cfb4ea.

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**